homeassistant/components/energy/data.py

```diff
@@ -25,6 +25,12 @@ async def async_get_manager(hass: HomeAssistant) -> EnergyManager:
     return manager
 
 
+async def is_configured(hass: HomeAssistant) -> bool:
```

Comment:
Let's move this to the base `__init__.py` module, so that other integrations can import it from the package and not need to know the modules.